### ISSUE TITLE
Allow labeling a task as expendable, to allow execution if those fail.

### ISSUE THREAD

**Issue by rbx** [FairRootGroup/ODC#69]
https://alice.its.cern.ch/jira/browse/EPN-131

**Comment by rbx** [comment]
https://alice.its.cern.ch/jira/browse/EPN-131

**Comment by rbx** [comment]
Available since ODC 0.76.0

To use them, add a custom DDS requirement whose name starts with `odc_expendable_` and value `true` to the DDS topology file.

Example:

```xml
<declrequirement name="odc_expendable_task" type="custom" value="true" />
```

And then use this requirement in the task declaration:
```xml
<decltask name="Processor">
  <exe>odc-ex-processor</exe>
  <requirements>
    <name>odc_expendable_task</name>
  </requirements>
</decltask>
```
